### PR TITLE
Refactor Instance and Meta Concept

### PR DESCRIPTION
**Pull request by fppt**
- Meta Concept is now Meta Thing
- Instance Class is not Thing

### CODE REVIEW

File: conf/test/cluster/grakn.properties [hunk 1]
```diff
@@ -54,7 +54,7 @@ tasks.postprocessing.delay=60000
 #Keyspace to be used when none is provided
 graphdatabase.default-keyspace=grakntest
 
-#A Type will be sharded when this instance threshold is hit
+#A Type will be sharded when this thing threshold is hit
```
Comment (aelred) [inline conf/test/cluster/grakn.properties]: hmm

**Comment by fppt** [comment]
@aelred I am glad I can entertain you :)

**Comment by fppt** [comment]
!rtg

File: grakn-core/src/main/java/ai/grakn/concept/Concept.java [hunk 2]
```diff
@@ -61,12 +61,12 @@
     Type asType();
 
     /**
-     * Return as an Instance if the Concept is an Instance.
+     * Return as an Thing if the Concept is an Thing.
```
Comment (aelred) [inline grakn-core/src/main/java/ai/grakn/concept/Concept.java]: There are a lot of cases of "an Thing" in the javadocs now.

**Pull request merged**